src/locales/en.json

```diff
@@ -92,7 +92,9 @@
     "alternate_names": "Alternate Names",
     "visible_to_public": "Visible to Public",
     "req_com_consent": "Requires Community Consent",
-    "create_dictionary": "Create Dictionary"
+    "create_dictionary": "Create Dictionary",
+    "permanent_url_msg": "The URL name is permanent and cannot be changed later.",
+    "add_gloss_msg": "You can add more glossing languages later in under Settings."
```

Comment:
@Danble, I think you are prepping ahead of time for adjusting the create-dictionary page. I think it's ok if we just tell them what can't be changed (the URL), and then let them make their way to the Settings page when needed by always making it easy to find. So this string can be deleted.